### PR TITLE
tidb-in-kubernetes: add tidb and tikv config hints

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add tidb and tikv config hints

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1767

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by dcalvin** [comment]
@juliezhang1112 please resolve the conflicts and fix the lint first. Thanks.

**Comment by juliezhang1112** [comment]
PTAL @dcalvin

File: dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ TiDB Operator uses `Helm` to deploy and manage TiDB clusters. The configuration
...
...lt TiKV configuration file (by choosing the tag of the corresponding TiKV version). You can see [TiKV Configuration Flags](/dev/reference/configuration/tikv-server/configuration.md) for the detailed description of the configuration parameters (by choosing the corresponding document version). Here you must **modify the configuration based on the format of the configuration file**.<br/><br/>You need to explicitly configure the following two configuration items:<br/><br/>`[storage.block-cache]`<br/>&nbsp;&nbsp;`shared = true`<br/>&nbsp;&nbsp;`capacity = "1GB"`<br/>It is recommended to set `capacity` to 50% of the value of `tikv.resources.limits.memory`.<br/><br/>`[readpool.coprocessor]`<br/>&nbsp;&nbsp;`high-concurrency = 8`<br/>&nbsp;&nbsp;`normal-concurrency = 8`<br/>&nbsp;&nbsp;`low-concurrency = 8`<br/>It is recommended to set to 80% of the value of `tikv.resources.limits.cpu`. | If the version of TiDB Operator is v1.0.0-beta.3 or earlier, the default value is<br>`nil`<br>If the version of TiDB Operator is later than v1.0.0-beta.3, the default value is<br>`log-level = "info"`<br>Example of configuration:<br>&nbsp;&nbsp;`config:` \|<br>&nbsp;&nbsp;&nbsp;&nbsp;`log-level = "info"` |
```
Comment (dcalvin) [inline dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]:
```suggestion
| `tikv.config` | The configuration of TiKV. Check [the `config-template.toml` file](https://github.com/tikv/tikv/blob/master/etc/config-template.toml) for the default TiKV configuration file (by choosing the tag of the corresponding TiKV version). You can see [TiKV Configuration Flags](/dev/reference/configuration/tikv-server/configuration.md) for the detailed description of the configuration parameters (by choosing the corresponding document version). Here you must **modify the configuration based on the format of the configuration file**.<br/><br/>You need to explicitly configure the following two configuration items:<br/><br/>`[storage.block-cache]`<br/>&nbsp;&nbsp;`shared = true`<br/>&nbsp;&nbsp;`capacity = "1GB"`<br/>It is recommended to set `capacity` to 50% of the value of `tikv.resources.limits.memory`.<br/><br/>`[readpool.coprocessor]`<br/>&nbsp;&nbsp;`high-concurrency = 8`<br/>&nbsp;&nbsp;`normal-concurrency = 8`<br/>&nbsp;&nbsp;`low-concurrency = 8`<br/>It is recommended to set to 80% of the value of `tikv.resources.limits.cpu`. | If the version of TiDB Operator is v1.0.0-beta.3 or earlier, the default value is<br>`nil`<br>If the version of TiDB Operator is later than v1.0.0-beta.3, the default value is<br>`log-level = "info"`<br>Sample configuration:<br>&nbsp;&nbsp;`config:` \|<br>&nbsp;&nbsp;&nbsp;&nbsp;`log-level = "info"` |
```

Comment (dcalvin) [reply dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]: Example of configuration -> Sample configuration

**Comment by juliezhang1112** [comment]
PTAL again, thanks. @dcalvin

**Pull request merged**